libs/compatibility-spy/src/index.ts

```diff
@@ -17,15 +18,26 @@ export class Platform implements platform.IPlatform {
    * @internal
    */
   public _usageContext: Map<string, Set<string>> = new Map();
+  private readonly jsii;
+  constructor() {
+    this.jsii = JSON.parse(
+      readFileSync(process.env.JSII_PATH ?? JSII_PATH, { encoding: "utf8" })
```

Comment:
I think this should work

```suggestion
      readFileSync(join(require.resolve(@winglang/sdk), "..", "..", ".jsii"), { encoding: "utf8" })
```

Reply:
Nice! it works thanks!!